services/contract_call_local.proto

```diff
@@ -132,6 +132,31 @@ message ContractFunctionResult {
      * EVM address described above.)
      */
      google.protobuf.BytesValue evm_address = 9;
+
+    /**
+     * The amount of gas that was used for this call. 
+     *
+     * This field should only be populated when the paired TransactionBody in the record stream is not a
+     * ContractCreateTransactionBody or a ContractCallTransactionBody.     
+     */
+    int64 gas = 10;
```

Comment:
Should we use wrapper types for these fields to clearly indicate not set or is primitive default values sufficient?

Reply:
There is no meaning for a missing field or a default field here like there is with the tracing valueWritten field, so a wrapper type would just add more bytes to the record.